pkg/operator/storageclasscontroller/storageclasscontroller.go

```diff
@@ -35,6 +35,9 @@ const (
 	resyncDuration        = 20 * time.Minute
 )
 
+// StorageClassController ultimately creates a StorageClass that
+// uses the vSphere CSI driver to provision volumes. On error,
+// this controller does not set any degraded condition.
```

Comment:
Which bug we are fixing? I thought the proper way of fixing this was to *not* create storageclass if we can't create storageclass. But if cluster meets OCP requirements of VCP CSI driver and if we can't create SC, we *SHOULD* degrade the cluster.

Reply:
Not a bug, but we discussed this multiple times during our design meetings. Specifically, this was agreed on on Sep 16 but we never created a card for it.